whois-commons/src/main/resources/nrtm_schema.sql

```diff
@@ -85,6 +86,19 @@ CREATE TABLE `notification_file`
 ) ENGINE = InnoDB
   DEFAULT CHARSET = utf8;
 
+
+CREATE TABLE `key_pair`
+(
+    `id`          int unsigned    NOT NULL AUTO_INCREMENT,
+    `private_key` VARBINARY(3000) NOT NULL,
+    `public_key`  VARBINARY(3000) NOT NULL,
+    `created`     bigint unsigned NOT NULL,
```

Comment:
Does it help to add an `expires` column rather than having to extract it from the key?

Reply:
I wouldn't assume a 1 year expiry either, we could expire more frequently in future (e.g. Let's Encrypt certificates expire after 90 days). The expiry column is a shortcut so you don't need to extract the expiry from the key itself, and allows you to query by expiry date (also useful for a monitoring check).

Reply:
we could create a new column called "end_date", which means that the key is not longer valid. 
From a job, we could calculate if the time ran out and modify that "end_date" and generate a new pair.

Reply:
I will add an expires column